web/packages/teleport/src/Roles/RoleEditor/standardmodel.ts

```diff
@@ -311,8 +368,9 @@ export function roleToRoleEditorModel(
  * Converts a `RoleConditions` instance (an "allow" or "deny" section, to be
  * specific) to a list of access specification models.
  */
-function roleConditionsToAccessSpecs(conditions: RoleConditions): {
+function roleConditionsToModel(conditions: RoleConditions): {
```

Comment:
Is this function supposed to return a specific subset of `RoleEditorModel`? If so, perhaps it would be a good idea to reuse that type here? E.g., `Pick<RoleEditorModel, 'accessSpecs' | 'rules' | 'requiresReset'>`.

The issue right now is that if I go to line 342 and do "Go to definition" on `requiresReset` to figure out what it means, I'm directed to line 374 instead of 62 which actually has a doc for this field.